tests/tasks_serverless.yml

```diff
@@ -0,0 +1,31 @@
+---
+requires:
+  serverless: true
+  stack: false
+---
+setup:
+  - do:
+      indices.create:
+        index: task_test
+---
+teardown:
+  - do:
+      indices.delete:
+        index: task_test
+        ignore: 404
+---
+"Task":
+  - do:
+      update_by_query:
+        index: task_test
+        wait_for_completion: false
+        body: {}
+  - set: { task: task }
+
+  - do:
+      tasks.get: { task_id: $task }
+      catch: resource_not_found_exception
```

Comment:
This fails for me:

```
AssertionError: Failed to catch 'resource_not_found_exception' in {'completed': True, ...}
```

I'm assuming the same applies to the stack test. Should we refactor the common part as well?

Reply:
Ouch, sorry, that was me not committing the right code! I've updated it to check for a `task` key which according to my tests *should* work even if `completed: false`.